EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -68,18 +85,18 @@ public struct MediaPlayerView: View {
         }
     }
     
-    @ViewBuilder private func renderForward() -> some View {
+    @ViewBuilder private func renderForward(disabled: Bool = false) -> some View {
         renderTimeElapsedWrapper {
-            renderButton(control: .forward)
+            renderButton(control: .forward, disabled: disabled)
```

Comment:
Any reason to apply this logic to forward control and not rewind?

Reply:
The forward button must be disabled once the video ends, but the rewind never gets disabled cause the user can still rewind it even if the video is over

Reply:
IMO if the user keeps pressing the rewind button, it should be disabled once there's nothing more to rewind. But if this is not the expected behavior/scope it's ok

Reply:
I see your point, but maybe this would be better fitted inside @LReiz task for the rewind and forward buttons itself... What do you think?